tasks/notify.py

```diff
@@ -328,6 +328,11 @@ def run_impl_within_lock(
                     test_result_commit_report is not None
                     and test_result_commit_report.test_result_totals is not None
```

Comment:
Could you change this one too?... Doesn't make sense to have "all_tests_passed" when actually no tests passed :E